### PR TITLE
Move Blackplum and Tenki Directory

### PR DESCRIPTION
**Pull request by eriqadams**
Move blackplum and tenki Directory into pixlupkeyboards directory.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/pixlupkeyboards/tenki/config.h [hunk 1]
```diff
@@ -0,0 +1,40 @@
+#pragma once
```
Comment (drashna) [inline keyboards/pixlupkeyboards/tenki/config.h:1]: Could you add a GPL2+ compatible license header here? 

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Or 

https://github.com/qmk/qmk_firmware/blob/6b299fab6e6a3143c061b9ea005253361fcd524b/data/templates/base/keyboard.c#L1-L15

File: keyboards/pixlupkeyboards/tenki/config.h [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+/* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
+#define LOCKING_SUPPORT_ENABLE
+
+/* Locking resynchronize hack */
+#define LOCKING_RESYNC_ENABLE
+
+
+/* ws2812 RGB LED */
+#define RGB_DI_PIN D1
+#define RGBLIGHT_ANIMATIONS
```
Comment (drashna) [inline keyboards/pixlupkeyboards/tenki/config.h:35]: Could you explicitly define which animations are enabled? 
```suggestion
#define RGBLIGHT_EFFECT_BREATHING
#define RGBLIGHT_EFFECT_RAINBOW_MOOD
#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
#define RGBLIGHT_EFFECT_SNAKE
#define RGBLIGHT_EFFECT_KNIGHT
#define RGBLIGHT_EFFECT_CHRISTMAS
#define RGBLIGHT_EFFECT_STATIC_GRADIENT
#define RGBLIGHT_EFFECT_RGB_TEST
#define RGBLIGHT_EFFECT_ALTERNATING
#define RGBLIGHT_EFFECT_TWINKLE
```

File: keyboards/pixlupkeyboards/tenki/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,19 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/pixlupkeyboards/tenki/keymaps/default/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Or 

https://github.com/qmk/qmk_firmware/blob/6b299fab6e6a3143c061b9ea005253361fcd524b/data/templates/base/keyboard.c#L1-L15

File: keyboards/pixlupkeyboards/tenki/keymaps/via/keymap.c [hunk 4]
```diff
@@ -0,0 +1,38 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/pixlupkeyboards/tenki/keymaps/via/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Or 

https://github.com/qmk/qmk_firmware/blob/6b299fab6e6a3143c061b9ea005253361fcd524b/data/templates/base/keyboard.c#L1-L15

File: keyboards/pixlupkeyboards/tenki/tenki.c [hunk 5]
```diff
@@ -0,0 +1 @@
+#include "tenki.h"
```
Comment (drashna) [inline keyboards/pixlupkeyboards/tenki/tenki.c:1]: Could you add a GPL2+ compatible license header here? 

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Or 

https://github.com/qmk/qmk_firmware/blob/6b299fab6e6a3143c061b9ea005253361fcd524b/data/templates/base/keyboard.c#L1-L15

File: keyboards/pixlupkeyboards/tenki/tenki.h [hunk 6]
```diff
@@ -0,0 +1,18 @@
+#pragma once
```
Comment (drashna) [inline keyboards/pixlupkeyboards/tenki/tenki.h:1]: Could you add a GPL2+ compatible license header here? 

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Or 

https://github.com/qmk/qmk_firmware/blob/6b299fab6e6a3143c061b9ea005253361fcd524b/data/templates/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/pixlupkeyboards/tenki/rules.mk [hunk 7]
```diff
@@ -0,0 +1,21 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = caterina
+
+# Build Options
+#   comment out to disable the options.
+#
+BOOTMAGIC_ENABLE = lite     # Enable Bootmagic Lite
```
Comment (drashna) [inline keyboards/pixlupkeyboards/tenki/rules.mk:10]: There is no "full" bootmagic anymore, only the "lite" option, which can be turned on with just "yes".
```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
```

**Review by drashna — APPROVED** [review]

File: keyboards/pixlupkeyboards/tenki/readme.md [hunk 8]
```diff
@@ -0,0 +1,16 @@
+# TENKI
+![TENKI](https://i.imgur.com/nOaxFwR.png)
+
+TENKI is a customizable hotswap mechanical numpad. 
+Supported layout:
+![TENKI LAYOUT](https://i.imgur.com/sD9kxID.png)
+
+Keyboard Maintainer: [eriqadams](https://github.com/eriqadams)  
+Hardware Supported: TENKI PCB, Pro Micro  
+Hardware Availability: [If you're in Indonesia you can purchase here](https://tokopedia.com/pixlup)
```
Comment (fauxpark) [inline keyboards/pixlupkeyboards/tenki/readme.md:10]:
```suggestion
* Keyboard Maintainer: [eriqadams](https://github.com/eriqadams)
* Hardware Supported: TENKI PCB, Pro Micro
* Hardware Availability: [If you're in Indonesia you can purchase here](https://tokopedia.com/pixlup)
```

File: keyboards/pixlupkeyboards/tenki/keymaps/default/keymap.c [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [0] = LAYOUT_ortho_5x4(
+    KC_NLCK, 	KC_PSLS,    KC_PAST,    	KC_PMNS,\
+    KC_P7,  	KC_P8,    	KC_P9,    		KC_PPLS,\
+    KC_P4,  	KC_P5,    	KC_P6,    		KC_BSPC,\
+    KC_P1,  	KC_P2,    	KC_P3,    		KC_PEQL,\
```
Comment (fauxpark) [inline keyboards/pixlupkeyboards/tenki/keymaps/default/keymap.c:11]:
```suggestion
    KC_NLCK, 	KC_PSLS,    KC_PAST,    	KC_PMNS,
    KC_P7,  	KC_P8,    	KC_P9,    		KC_PPLS,
    KC_P4,  	KC_P5,    	KC_P6,    		KC_BSPC,
    KC_P1,  	KC_P2,    	KC_P3,    		KC_PEQL,
```

File: keyboards/pixlupkeyboards/tenki/keymaps/default/keymap.c [hunk 10]
```diff
@@ -0,0 +1,22 @@
...
+    KC_P0, 	KC_PSCR , 	LT(1,KC_PDOT),  KC_PENT
+  ),
+
+  [1] = LAYOUT_ortho_5x4(
+    KC_TRNS, 	RGB_HUI,    RGB_HUD,    	RGB_VAD,\
+    KC_TRNS, 	KC_VOLU,    RGB_TOG,    	RGB_VAI,\
+    KC_MPRV, 	KC_MUTE,    KC_MNXT,    	KC_TRNS,\
+    KC_TRNS,  KC_VOLD,    KC_TRNS,    	KC_TRNS,\
```
Comment (fauxpark) [inline keyboards/pixlupkeyboards/tenki/keymaps/default/keymap.c:19]:
```suggestion
    KC_TRNS, 	RGB_HUI,    RGB_HUD,    	RGB_VAD,
    KC_TRNS, 	KC_VOLU,    RGB_TOG,    	RGB_VAI,
    KC_MPRV, 	KC_MUTE,    KC_MNXT,    	KC_TRNS,
    KC_TRNS,  KC_VOLD,    KC_TRNS,    	KC_TRNS,
```

File: keyboards/pixlupkeyboards/blackplum/rules.mk [hunk 11]
```diff
@@ -7,7 +7,7 @@ BOOTLOADER = qmk-dfu
 # Build Options
 #   comment out to disable the options.
```
Comment (fauxpark) [inline keyboards/pixlupkeyboards/blackplum/rules.mk:8]:
```suggestion
#   change yes to no to disable
```

File: keyboards/pixlupkeyboards/tenki/keymaps/via/keymap.c [hunk 12]
```diff
@@ -0,0 +1,41 @@
...
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [0] = LAYOUT_ortho_5x4(
+    KC_NLCK, 	KC_PSLS,    KC_PAST,    	KC_PMNS,\
+    KC_P7,  	KC_P8,    	KC_P9,    		KC_PPLS,\
+    KC_P4,  	KC_P5,    	KC_P6,    		KC_BSPC,\
+    KC_P1,  	KC_P2,    	KC_P3,    		KC_PEQL,\
```
Comment (fauxpark) [inline keyboards/pixlupkeyboards/tenki/keymaps/via/keymap.c:11]:
```suggestion
    KC_NLCK, 	KC_PSLS,    KC_PAST,    	KC_PMNS,
    KC_P7,  	KC_P8,    	KC_P9,    		KC_PPLS,
    KC_P4,  	KC_P5,    	KC_P6,    		KC_BSPC,
    KC_P1,  	KC_P2,    	KC_P3,    		KC_PEQL,
```

File: keyboards/pixlupkeyboards/tenki/keymaps/via/keymap.c [hunk 13]
```diff
@@ -0,0 +1,41 @@
...
+    KC_P0, 	KC_PSCR, 	LT(1,KC_PDOT),  KC_PENT
+  ),
+
+  [1] = LAYOUT_ortho_5x4(
+    KC_TRNS, 	RGB_HUI,    RGB_HUD,    	RGB_VAD,\
+    KC_TRNS, 	KC_VOLU,    RGB_TOG,    	RGB_VAI,\
+    KC_MPRV, 	KC_MUTE,    KC_MNXT,    	KC_TRNS,\
+    KC_TRNS,  KC_VOLD,    KC_TRNS,    	KC_TRNS,\
```
Comment (fauxpark) [inline keyboards/pixlupkeyboards/tenki/keymaps/via/keymap.c:19]:
```suggestion
    KC_TRNS, 	RGB_HUI,    RGB_HUD,    	RGB_VAD,
    KC_TRNS, 	KC_VOLU,    RGB_TOG,    	RGB_VAI,
    KC_MPRV, 	KC_MUTE,    KC_MNXT,    	KC_TRNS,
    KC_TRNS,  KC_VOLD,    KC_TRNS,    	KC_TRNS,
```

File: keyboards/pixlupkeyboards/tenki/keymaps/via/keymap.c [hunk 14]
```diff
@@ -0,0 +1,41 @@
...
+    RESET,    KC_TRNS,    KC_TRNS,        RGB_MOD
+  ),
+  
+  [2] = LAYOUT_ortho_5x4(
+    KC_TRNS, 	KC_TRNS,    KC_TRNS,    	KC_TRNS,\
+    KC_TRNS, 	KC_TRNS,    KC_TRNS,    	KC_TRNS,\
+    KC_TRNS, 	KC_TRNS,    KC_TRNS,    	KC_TRNS,\
+    KC_TRNS, 	KC_TRNS,    KC_TRNS,    	KC_TRNS,\
```
Comment (fauxpark) [inline keyboards/pixlupkeyboards/tenki/keymaps/via/keymap.c:27]:
```suggestion
    KC_TRNS, 	KC_TRNS,    KC_TRNS,    	KC_TRNS,
    KC_TRNS, 	KC_TRNS,    KC_TRNS,    	KC_TRNS,
    KC_TRNS, 	KC_TRNS,    KC_TRNS,    	KC_TRNS,
    KC_TRNS, 	KC_TRNS,    KC_TRNS,    	KC_TRNS,
```

File: keyboards/pixlupkeyboards/tenki/keymaps/via/keymap.c [hunk 15]
```diff
@@ -0,0 +1,41 @@
...
+    KC_TRNS, 	KC_TRNS,    KC_TRNS,    	KC_TRNS
+  ),
+  
+  [3] = LAYOUT_ortho_5x4(
+    KC_TRNS, 	KC_TRNS,    KC_TRNS,    	KC_TRNS,\
+    KC_TRNS, 	KC_TRNS,    KC_TRNS,    	KC_TRNS,\
+    KC_TRNS, 	KC_TRNS,    KC_TRNS,    	KC_TRNS,\
+    KC_TRNS, 	KC_TRNS,    KC_TRNS,    	KC_TRNS,\
```
Comment (fauxpark) [inline keyboards/pixlupkeyboards/tenki/keymaps/via/keymap.c:35]:
```suggestion
    KC_TRNS, 	KC_TRNS,    KC_TRNS,    	KC_TRNS,
    KC_TRNS, 	KC_TRNS,    KC_TRNS,    	KC_TRNS,
    KC_TRNS, 	KC_TRNS,    KC_TRNS,    	KC_TRNS,
    KC_TRNS, 	KC_TRNS,    KC_TRNS,    	KC_TRNS,
```

File: keyboards/pixlupkeyboards/tenki/rules.mk [hunk 16]
```diff
@@ -0,0 +1,21 @@
...
+BOOTMAGIC_ENABLE = yes    # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = yes   # Mouse keys
+EXTRAKEY_ENABLE = yes   # Audio control and System control
+CONSOLE_ENABLE = no     # Console for debug
+COMMAND_ENABLE = no     # Commands for debug and configuration
+SLEEP_LED_ENABLE = no   # Breathing sleep LED during USB suspend
+NKRO_ENABLE = yes       # USB Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+BACKLIGHT_ENABLE = no   # Enable keyboard backlight functionality
+AUDIO_ENABLE = no
+RGBLIGHT_ENABLE = yes
```
Comment (fauxpark) [inline keyboards/pixlupkeyboards/tenki/rules.mk:19]:
```suggestion
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = yes           # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
AUDIO_ENABLE = no           # Audio output
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/pixlupkeyboards/tenki/rules.mk [hunk 17]
```diff
@@ -0,0 +1,21 @@
...
```
Comment (fauxpark) [inline keyboards/pixlupkeyboards/tenki/rules.mk:19]:
```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = yes           # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
AUDIO_ENABLE = no           # Audio output
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by drashna** [comment]
This has merge conflicts.

**Comment by eriqadams** [comment]
> This has merge conflicts.

Ok

**Comment by tzarc** [comment]
Moves need to go through the `develop` branch to minimise disruption to users.

File: keyboards/pixlupkeyboards/tenki/rules.mk [hunk 18]
```diff
@@ -0,0 +1,25 @@
...
+#
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
```
Comment (fauxpark) [inline keyboards/pixlupkeyboards/tenki/rules.mk:18]:
```suggestion
NKRO_ENABLE = yes           # Enable N-Key Rollover
```

File: keyboards/pixlupkeyboards/tenki/rules.mk [hunk 19]
```diff
@@ -0,0 +1,25 @@
...
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+AUDIO_ENABLE = no           # Audio output
+
+
```
Comment (fauxpark) [inline keyboards/pixlupkeyboards/tenki/rules.mk:23]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Pull request closed without merging**